tests/shell/include_common.sh.in

```diff
@@ -60,6 +60,12 @@ then
 	KDB=${KDB_GETENV};
 fi
 
+if [[ "$OSTYPE" == "darwin"* ]]; then
+	TMPPATH="/private/tmp"
+else
+	TMPPATH="/tmp"
+fi
+
```

Comment:
This file needs to have 1000 lines for correct line number, either you reduce empty lines at the end, or simply move that code to check_resolver.sh, because it is not used somewhere else anyway.